README.md

```diff
@@ -166,6 +166,8 @@ ESLint follows [semantic versioning](http://semver.org). However, due to the nat
     * An existing formatter is removed.
     * Part of the public API is removed or changed in an incompatible way.
 
+Because our policy allows a minor release to result in reporting more errors, it can be seen as a "breaking change". Thus if you don't want an update to ESLint to break your lint build, we recommend you you `~` in `package.json` e.g. `"eslint": "~3.1.0"` in order to only get patch changes.
```

Comment:
Let's not use the term "breaking change" here, as I think it will confuse people. It's better to say that any minor update may include more warnings than the previous release. As such, we recommend using the tilde if you want to guarantee the results of your builds.